### ISSUE TITLE
-baker-rendering optional in hpub, but for Baker app?

### ISSUE THREAD

**Issue by psmedia** [Simbul/baker#200]
Small comment here. 

I left out the -baker-rendering entry in my book.json file, and had frequent memory warnings and hard to pin-down crashes in my app. On specifying the

 -baker-rendering: "screenshot", 

in the book.json file , performance improved markedly! Not sure why this would be the case, but perhaps documentation should stress inclusion of this parameter in the book.json file for Baker apps, even if it is (of course) optional for hpub.

**Comment by Simbul** [comment]
This is extremely weird: when the `-baker-rendering` property is not specified, it is set by default to `screenshot`.

Are you sure the improvement in performance cannot be attributed to some other change? Have you tried going back to a manifest without an explicit `-baker-rendering` to see whether the performance issues come back?

Sorry about all the questions, but as you can imagine this kind of issue is very weird and difficult for us to reproduce ;)

**Comment by psmedia** [comment]
Yes, I saw that note in the documentation, but was so struck by the difference in performance...

This was the only change I made in the application, but I may have changed something in the test iPad. I will go back and see if I can reproduce the errors. 

I _totally_ understand, and hesitated to share this because I knew it was likely to be specific to my app/dev't setup. But then thought there might something in the way the app implements the default setting? Anyway, I will let you know the results of my attempts to reproduce the error.

Thanks as always!
